### ISSUE TITLE
How to get pretrain model?

### ISSUE THREAD

**Issue by Laudarisd** [AlvaroCavalcante/auto_annotate#13]
Hello Alvaro, thanks for the nice tool. 
I am trying to use your idea. 

Would you kindly let me know the way to get pretrained model?
Thanks.

One more thing, so tf model folder and auto_annotate stays separately of do i have to put tf model inside of auto_annotate?

**Comment by Laudarisd** [comment]
I think I solved this problem. Thanks

**Comment by AlvaroCavalcante** [comment]
Ok, If you need help, just open another issue, I'll be glad to help you! :)

**Comment by Vibha111094** [comment]
Hi, 
I would be really interested to know how to get the pre-trained model.
Is it like you label a few images and fine-tune them on an object detection model like YOLO?

**Comment by Laudarisd** [comment]
@Vibha111094  In my understanding, pre trained models are easily available in open sources. 
Check here : https://github.com/tensorflow/tfjs-models.

If you are thinking to make your own pre trained model them then its going to be time consuming and challenging.

This might give you simple idea about making resnet model in keras: https://towardsdatascience.com/hitchhikers-guide-to-residual-networks-resnet-in-keras-385ec01ec8ff 

Even training YOLO usages pre trained mode.
I hope I answered your question.

**Comment by Vibha111094** [comment]
@Laudarisd thankyou for your reply.
However, I am not very clear what is the data that the pre-trained model is trained on (for this particular use case)
